Use omiconfigeditor to determine state of port 1270

vimish opened the pull request:
@Microsoft/omsagent-devs 
@jeffaco 

omiconfigeditor now supports query option. Existing
logic, to determine if port 1270 is open or not, is
updated to use omiconfigeditor.

PBUILD has passed for the changes.

jeffaco commented on installer/scripts/omsadmin.sh:
```diff
@@ -445,8 +428,8 @@ onboard_scom()
-        /opt/omi/bin/omiconfigeditor httpsport -a 1270 < /etc/opt/omi/conf/omiserver.conf > /etc/opt/omi/conf/omiserver.conf_temp
-        mv /etc/opt/omi/conf/omiserver.conf_temp /etc/opt/omi/conf/omiserver.conf
+        $OMI_CONF_EDITOR httpsport -a 1270 < $OMI_CONF_FILE > /etc/opt/omi/conf/omiserver.conf_temp
+        mv /etc/opt/omi/conf/omiserver.conf_temp $OMI_CONF_FILE
```
This could be rewritten as:

```
$OMI_CONF_EDITOR httpsport -a 1270 < $OMI_CONF_FILE > ${OMI_CONF_FILE}_temp
mv ${OMI_CONF_FILE}_temp $OMI_CONF_FILE
```

Just another way to do the same thing. I'm not asking for the change, just pointing it out.

lagalbra commented on installer/scripts/omsadmin.sh:
```diff
@@ -95,6 +95,7 @@ PROC_LIMIT_CONF=/etc/security/limits.conf
 # SCOM variables
 SCX_SSL_CONFIG=/opt/microsoft/scx/bin/tools/scxsslconfig
 OMI_CONF_FILE=/etc/opt/omi/conf/omiserver.conf
+OMI_CONF_EDITOR=/opt/omi/bin/omiconfigeditor
```
@jeffaco Is there a minimum version of OMI that supports this? If this was only added in a recent version, then we may want to update our dependencies: https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/datafiles/linux_rpm.data#L10 and https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/datafiles/linux_dpkg.data#L9

jeffaco replied:
That is true. @vimish added this to OMI v1.3. So OMS now has a minimum dependency of OMS v1.3 since it depends on this capability. @vimish, I'm going to mark my review "Waiting" for this change.

vimish commented:
@jeffaco @lagalbra 

Updated the min version of omi to 1.3.0-2.
Also updated min version of scx to 1.6.3-212 as omsagent bundle supports --enable-opsmgr flag. This flag is introduced in scx 1.6.3-212 bundle.

lagalbra commented:
@jeffaco Can you approve so that Vishal can merge? I've tested this briefly myself, and it seems to work.
